### PR TITLE
Use custom image

### PR DESCRIPTION
**Pull request by trey-stafford**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ dependencies:
   - parsl-with-kubernetes ~=2024.4
   # TODO: Remove once this build is completed: https://github.com/conda-forge/parsl-feedstock/pull/72
   - python-kubernetes
+
+  - pip:
+      - "--editable=git+https://github.com/QGreenland-Net/parsl.git@k8s-use-incluser-config-fallback#egg=parsl"
```
Comment (mfisher87) [inline environment.yml:17]: We should probably explain this and link to the PR in a comment

File: environment.yml [hunk 2]
```diff
@@ -7,3 +7,6 @@ dependencies:
   - parsl-with-kubernetes ~=2024.4
   # TODO: Remove once this build is completed: https://github.com/conda-forge/parsl-feedstock/pull/72
   - python-kubernetes
```
Comment (mfisher87) [inline environment.yml]: I think we can follow through on this comment now too

**Review by mfisher87 — APPROVED** [review]

**Review by rushirajnenuji — APPROVED** [review]

File: environment.yml (hunk 1 above)
Comment (mfisher87) [reply environment.yml:17]: 86b5c23

File: environment.yml (hunk 2 above)
Comment (mfisher87) [reply environment.yml]: 7d94f81

**Pull request merged**